agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/BeforeAgentInstaller.java

```diff
@@ -125,6 +126,7 @@ private static void start(Instrumentation instrumentation) throws Exception {
         // TODO need some kind of test for these configuration properties
         if (!isInstrumentationEnabled(config, "micrometer")) {
             properties.put("otel.instrumentation.micrometer.enabled", "false");
```

Comment:
look like micrometer is disabled already... this is for disabling actuator-metrics.  is actuator-metrics part of micrometer?

Reply:
they are both for capturing micrometer metrics, but in different ways:

"micrometer" instrumentation captures micrometer metrics captured via the global micrometer api

"actuator-metrics" instrumentation captures micrometer metrics captured via spring boot actuator metrics